src/RocksDBStore/Plugins/Storage/RocksDBStorageProvider.cs

```diff
@@ -0,0 +1,19 @@
+using Neo.Persistence;
+
+namespace Neo.Plugins.Storage
+{
+    public class RocksDBStorageProvider : Plugin, IStorageProvider
```

Comment:
The plugin name should not be changed. Otherwise it will cause problems in the cli.